README.md

```diff
@@ -1,6 +1,7 @@
 # InteractiveProgramming
+This is a two player pong game with Pikachus! 
+(1) Download the three images (pokeball, pokemon, grass)
+(2) Control Pikachu 1 on the left using the 's' key to move up and 'x' key to move down. Control Pikachu 2 on the right using the up arrow key to move up and down arrow key to move down.
 
-This is the base repo for the interactive programming project for Software Design, Fall 2016 at Olin College.
-
-This file is not a good project README.
-Replace the contents of this file before you submit your project.
\ No newline at end of file
+Instructions:
+Prevent the ball from hitting the wall on your side by moving Pikachu and using it to bounce the ball back. If the ball hits the wall on your side, your opponent gets a point. The first Pikachu/player to accumulate five points will win the game!
```

Comment:
Y'all don't mention quite how to start the program/what the program's requirements are. (mentioning how to run the program, e.g. something as simple as saying that once python + pygame are installed you can call `python pokemon_pong.py`, would be good)